tracerite/style.css

```diff
@@ -135,3 +167,28 @@
   border-collapse: collapse;
   text-align: right;
 }
+
+[data-tooltip] {
+  position: relative;
+}
+
+[data-tooltip]::before {
+  position: absolute;
+  content: attr(data-tooltip);
+  opacity: 0;
+  color: var(--tracerite-label);
+  background: var(--tracerite-dark);
+  top: 50%;
+  transform: translate(2ch, -50%);
+  right: 0;
+  font-size: 0.8em;
+  padding: 0.5em 1em;
+  border-radius: 3px;
+  text-align: center;
+  text-transform: capitalize;
+  z-index: 2;
+}
+
+[data-tooltip]:hover::before {
+  opacity: 1;
+}
```

Comment:
Could the tooltip be hidden with `display: none` or `visibility: hidden` instead or did none of those work?

Reply:
sure, any of those work